unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Tests/BIWRotateGizmosShould.cs

```diff
@@ -0,0 +1,80 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+public class BIWRotateGizmosShould
+{
+    private GameObject mockedGizmosGameObject;
+    private GameObject mockedEntityGameObject;
+    private BIWGizmosRotate gizmos;
+    private BIWGizmosAxis axis;
+
+    [UnitySetUp]
+    protected IEnumerator SetUp()
```

Comment:
As you are not using the coroutine features, you can just do:
```
[SetUp]
void SetUp()
{
    ...
}
``` 
And the same for the `TearDown` method.